### ISSUE TITLE
Data.json not found on Mac OS X

### ISSUE THREAD

**Issue by frankhale** [frankhale/toby#16]
Need to figure out how to properly set up the path so that it works on Windows, Linux and Mac

```
Uncaught Error: ENOENT: no such file or directory, open '//resources/app/data/data.json'
```

**Comment by frankhale** [comment]
This works on OSX but does it work on Windows? I'm about to find out...

```
dataFilePath: __dirname + path.sep + ["data", "data.json"].join(path.sep)
```

**Comment by frankhale** [comment]
Fixed via 3e9d398097a1612d808df8cc1fcd541197aa6373
